gulpfile.js

```diff
@@ -113,7 +113,10 @@ gulp.task('build-watch', function() {
 });
 
 gulp.task('buildApps', function() {
-    return buildCesiumViewer();
+    return buildCesiumViewer()
```

Comment:
Any reason we can't build these in parallel via a `Promise.join` instead of then?